logic.typ

```diff
@@ -249,14 +249,14 @@
 #let pause = {
   // We need two separate `locate`s because `repetitions` needs to be updated
   // using the new value of `pause-counter`.
-  locate( loc => {
-    if not handout-mode.at(loc) {
+  context {
+    if not handout-mode.get() {
       pause-counter.step()
     }
-  })
-  locate( loc => {
-    repetitions.update(rep => calc.max(rep, pause-counter.at(loc).first() + 1))
-  })
+  }
+  context {
+    repetitions.update(rep => calc.max(rep, pause-counter.get().first() + 1))
```

Comment:
Calling `.get()` in this closure does not work, since the context is not available in it.

Alternative:
```typst
let pauses = pause-counter.get().first() + 1
repetitions.update(rep => calc.max(rep, pauses))
```